source/javascripts/controllers/_AndroidKeystoreController.js.erb

```diff
@@ -74,6 +70,10 @@ angular.module("BitriseWorkflowEditor").controller("AndroidKeystoreController",
 				viewModel.keystoreFile = null;
 			}
 
+			viewModel.showPassword = viewModel.keystoreFile ? _.isUndefined(viewModel.keystoreFile.password) : false;
```

Comment:
These should only be done in the case when there is already a file uploaded. It is the block starting at line 46 `if (data !== null) {`.

For the case when there is no uploaded file yet, but user later uploads one, do this same thing in the upload method's success block.

Also, do this same thing after saving the metadata - see other comment below.

And it does not need to be checked whether it is undefined or not, simply `!viewModel.keystoreFile.password` should be enough, so it handles empty strings as well.